c_api/IndexIVF_c.h

```diff
@@ -114,6 +114,8 @@ double faiss_IndexIVF_imbalance_factor (const FaissIndexIVF* index);
 /// display some stats about the inverted lists of the index
 void faiss_IndexIVF_print_stats (const FaissIndexIVF* index);
 
+/// get inverted lists ids
```

Comment:
I would prefer to document this function better, as it could make room for misuse. In particular, the user needs to first call `faiss_IndexIVF_get_list_size` to ensure that a large enough array is allocated, and only then call this function.

```suggestion
/// Get the IDs in an inverted list. IDs are written to `invlist`, which must be large enough
//// to accommodate the full list.
///
/// @param list_no the list ID
/// @param invlist output pointer to a slice of memory, at least as long as the list's size
/// @see faiss_IndexIVF_get_list_size(size_t) 
```